src/dymoprint/lib/render_engines/print_payload.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from PIL import Image
+
+from dymoprint.lib.render_engines.margins import MarginsMode, MarginsRenderEngine
+from dymoprint.lib.render_engines.render_context import RenderContext
+from dymoprint.lib.render_engines.render_engine import RenderEngine
+
+
+class PrintPayloadRenderEngine(RenderEngine):
+    def __init__(
+        self,
+        render_engine: RenderEngine,
+        justify: str = "center",
```

Comment:
```suggestion
        justify: Literal["left", "center", "right"] = "center",
```

Reply:
There's still one missing in `dymoprint.gui.q_dymo_labels_list` line 116.

Reply:
Fixed.
That's exactly what I dislike about working wet, without Enums... You have to update 100 places.
I've searched in the code, and I think I've updated in all places.

Reply:
Seems to me like a feature. Sometimes a change does require an update in 100 places, and this makes it explicit. :slightly_smiling_face: